cms/templates/ux/reference/modal_access-component.html

```diff
@@ -0,0 +1,126 @@
+                            <label for="visibility-content-group-NAME3" class="label">Content Group NAME 3</label>
+                          </li>
+
+                          <li class="field field-checkbox field-visibility-content-group">
+                            <input type="checkbox" id="visibility-content-group-NAME4" name="visibility-content-group" value="" class="input input-checkbox visibility-content-group-NAME4" />
+                            <label for="visibility-content-group-NAME4" class="label">Content Group NAME 4</label>
+                          </li>
+
+                          <!-- NOTE: @andyarmstrong, here's an example of how we would treat a group that was deleted/removed - we need a .was removed class and an additional UI element called a .note -->
+                          <li class="field field-checkbox field-visibility-content-group was-removed">
+                            <input type="checkbox" id="visibility-content-group-deleted1" name="visibility-content-group" value="" class="input input-checkbox visibility-content-group-deleted" checked="checked" />
+                            <label for="visibility-content-group-deleted1" class="label">
+                              Deleted Content Group
+                            </label>
+                            <span class="note">The selected group no longer exists. Choose another group or make the component visible to All Students and Staff</span>
+                          </li>
+
+                          <!-- NOTE: @andyarmstrong, here's an example of how we would treat a group that was deleted/removed - we need a .was removed class and an additional UI element called a .note -->
+                          <li class="field field-checkbox field-visibility-content-group was-removed">
+                            <input type="checkbox" id="visibility-content-group-deleted1" name="visibility-content-group" value="" class="input input-checkbox visibility-content-group-deleted" checked="checked" />
+                            <label for="visibility-content-group-deleted1" class="label">
+                              Deleted Content Group
+                            </label>
+                            <span class="note">The selected group no longer exists. Choose another group or make the component visible to All Students and Staff</span>
+                          </li>
+                        </ul>
+                      </div>
+                    </section>
+                  </div>
+                </div>
+              </div>
+
+            </form>
+
+          </div><!-- .xblock -->
+        </div><!-- .xblock-editor -->
+      </div><!-- .modal-content -->
+
+      <div class="modal-actions">
+        <h3 class="sr">Actions</h3>
+        <ul>
+          <li class="action-item">
+            <a href="#" class="button action-primary action-save">Save</a>
```

Comment:
these should be buttons (or have role="button") since they are associated with submitting a form.